src/sentry/metrics/dogstatsd.py

```diff
@@ -110,9 +110,21 @@ def distribution(
         sample_rate: float = 1,
         unit: str | None = None,
         stacklevel: int = 0,
+        precise: bool = False,
     ) -> None:
-        # We keep the same implementation for Datadog.
-        self.timing(key, value, instance, tags, sample_rate)
```

Comment:
well, I’m wondering if this might be one contributing factor why our distributions are as bad as they are 🤔